src/rake.rs

```diff
@@ -26,7 +26,16 @@ impl Rake {
     /// Runs RAKE algorithm on `text` and returns a vector of keywords.
     /// The returned vector is sorted by score (from greater to less).
     pub fn run(&self, text: &str) -> Vec<KeywordScore> {
```

Comment:
please add `#[inline]` for this function, too.

Reply:
It doesn't use monomorphisation, so it's unlikely to benefit from that.